Rhinoceros_Engine/Convert/ToRhino.cs

```diff
@@ -610,6 +620,46 @@ public static List<object> ToRhino(this BHG.CompositeGeometry geometries)
             return geometries.Elements.Select(x => x.IToRhino()).ToList();
         }
 
+        /***************************************************/
+        /**** Representation  fallback                  ****/
+        /***************************************************/
+
+        public static object ToRhino(this IRender render)
```

Comment:
This should be private, I think. We should not expose fallback in the UI.

Reply:
If you agree with the above, this method should be moved among the private methods section below the public ones. The header should read "Private methods - fallback"

Reply:
Having had a better look, I can't seem to find any fallback for `ToRhino(IGeometry ...)`!

I think our approach could be to add a single fallback that accepts `IObject` so we cover both `IGeometry` and `IRender`.